crates/sqlbuiltins/src/functions/table/cassandra.rs

```diff
@@ -0,0 +1,65 @@
+use std::collections::HashMap;
+use std::sync::Arc;
+
+use async_trait::async_trait;
+use datafusion::arrow::datatypes::DataType;
+use datafusion::datasource::TableProvider;
+use datafusion::logical_expr::{Signature, Volatility};
+use datafusion_ext::errors::{ExtensionError, Result};
+use datafusion_ext::functions::{FuncParamValue, TableFuncContextProvider};
+use datasources::cassandra::CassandraTableProvider;
+use protogen::metastore::types::catalog::{FunctionType, RuntimePreference};
+
+use super::TableFunc;
+use crate::functions::ConstBuiltinFunction;
+
+#[derive(Debug, Clone, Copy)]
+pub struct ReadCassandra;
+
+impl ConstBuiltinFunction for ReadCassandra {
+    const NAME: &'static str = "read_cassandra";
+    const DESCRIPTION: &'static str = "Read a Cassandra table";
+    const EXAMPLE: &'static str = "SELECT * FROM read_cassandra('localhost:9000', '<ns>.<table>')";
```

Comment:
```suggestion
    const EXAMPLE: &'static str = "SELECT * FROM read_cassandra('localhost:9000', '<ns>', '<table>')";
```

Key space and table should be provided as separate args for consistency with our other reads. Longer term we can look into allowing the qualified '<schema>.<table>' form, but I'd want to do that wholesale for all the reads.